Makefile

```diff
@@ -138,15 +138,25 @@ endif
 endif
 
 # ---------------------------------------------------------------------
+# Electron Builder
+# ---------------------------------------------------------------------
+
+ELECTRON_BUILDER_OPTIONS=--$(TARGET_ARCH_ELECTRON_BUILDER) --extraMetadata.version=$(APPLICATION_VERSION)
```

Comment:
Ah, I totally misunderstood the comment. That works fine, I'm updating the PR :)